porcelain/wallet.go

```diff
@@ -27,3 +32,14 @@ func WalletBalance(ctx context.Context, plumbing walletPlumbing, addr address.Ad
 
 	return act.Balance, nil
 }
+
+// DefaultWalletAddress returns a default wallet address from the config.
+func DefaultWalletAddress(plumbing walletPlumbing) (address.Address, error) {
+	ret, err := plumbing.ConfigGet("wallet.defaultAddress")
+	addr := ret.(address.Address)
+	if err != nil || addr != (address.Address{}) {
+		return addr, err
+	}
+
+	return address.Address{}, ErrNoDefaultWalletAddress
```

Comment:
If no address is present in the config, we should attempt to set one using the first available address in the wallet store. Check out the `GetAndMaybeSetDefaultSenderAddress` method in `porcelain/message.go` around line 60. I believe this method does what you're looking for here.

This method has a better name, so you could move that method's logic here, delete the old one, and replace any invocations of `GetAndMaybeSetDefaultSenderAddress` with `DefaultWalletAddress `.

Reply:
Ok, thank you, I'll fix it tomorrow.

Reply:
Thanks!

Reply:
I've fix it just now, please let me know if there is any problems, thanks!